src/index.js

```diff
@@ -38,11 +42,15 @@ function fetchAndLoadDataModel() {
     .then(responses => {
       if (checkLoginCredentials(responses[0], username, password, id)) {
         generateClasses(responses[0], responses[1], responses[2]);
+        displayAllTrips();
+        domUpdates.displaySidebar(user, sidebar);
```

Comment:
Possibly try to break this code into smaller functions that perform a specific login display page function?